client/scss/components/forms/_input-text.scss

```diff
@@ -38,6 +38,14 @@ textarea {
 .w-field--date_field,
 .w-field--date_time_field,
 .w-field--time_field {
+  // If these fields are in an inline panels' row then allow them to take up their grid size
+  .w-field-row & {
+    input {
+      width: 100%;
+      max-width: 250px;
```

Comment:
Should we use rem here? Or is there an existing variable we can use for this width?

Reply:
Yep I can switch this to use rems for sure. I haven't been able to find any max input width anywhere though.